packages/shared/src/components/ProfileMenu/ProfileMenuHeader.tsx

```diff
@@ -46,6 +60,16 @@ export const ProfileMenuHeader = (): ReactElement => {
           @{user.username}
         </Typography>
       </div>
+
+      {shouldOpenProfile && (
+        <>
+          <OpenLinkIcon className="text-text-quaternary" />
+          <Link href={`${webappUrl}${user.username}`} passHref>
+            {/* eslint-disable-next-line jsx-a11y/anchor-has-content */}
```

Comment:
if you don't use `passHref` then you should be able to have empty link maybe?

Reply:
Refactored it so that the anchor wraps around instead